app/views/PaymentAllocation.scala.html

```diff
@@ -88,10 +85,13 @@
         </div>
     </dl>
 
-@govukInsetText(InsetText(content = HtmlContent(paymentAllocationInset)))
+@p(id=Some("paymentAllocation-intro")) {@messages("paymentAllocation.intro")}
+
+@numberList("paymentAllocation.condition-1", "paymentAllocation.condition-2", "paymentAllocation.condition-3")
 
-    @if(paymentAllocations.paymentAllocationChargeModel.documentDetails.exists(_.outstandingAmountZero) && CutOverCreditsEnabled && paymentAllocations.paymentAllocationChargeModel.documentDetails.exists(_.credit.isDefined)) {
+@p(id=Some("paymentAllocation-outro")) {@messages("paymentAllocation.outro")}
 
+@if(paymentAllocations.paymentAllocationChargeModel.documentDetails.exists(_.outstandingAmountZero) && CutOverCreditsEnabled && paymentAllocations.paymentAllocationChargeModel.documentDetails.exists(_.credit.isDefined)) {
```

Comment:
* I am not sure why we have if block, as we don't have anything to render inside it; if you can double check this logic;
* if you need these conditions can we move these on the level of viewModel;